volmdlr/edges.py

```diff
@@ -5925,27 +5851,25 @@ def theta_A_B(s, i, e, c):
 
         if start == end:
             extra_new = frame.global_to_local_coordinates(self.interior)
-            theta, A, B = theta_A_B(start_new, extra_new, interior_new,
-                                    center_new)
+            theta, major_axis, minor_axis = theta_a_b(start_new, extra_new, interior_new, center_new)
         else:
-            theta, A, B = theta_A_B(start_new, interior_new, end_new,
-                                    center_new)
+            theta, major_axis, minor_axis = theta_a_b(start_new, interior_new, end_new, center_new)
 
-        self.Gradius = A
-        self.Sradius = B
+        self.Gradius = major_axis
+        self.Sradius = minor_axis
```

Comment:
Why not change self.Gradius to self.major_axis and self.Sradius to self.minor_axis as it's in other ellipse objects?

Reply:
Yes, indeed. I've considered it. I just thought it would be a better ideia to change it in another branch. I am afraid we're going to change it in a lot of places.